README.md

```diff
@@ -22,6 +22,51 @@ And this will be your templates after you use Mustache:
 
 !['stache](https://cloud.githubusercontent.com/assets/288977/8779228/a3cf700e-2f02-11e5-869a-300312fb7a00.gif)
 
+## Install
+
+You can get Mustache on npm.
+
+```bash
+$ npm install mustache --save
+```
+or install with bower:
+
+```bash
+$ bower install --save mustache
+```
+
+## Command line tool
+
+mustache.js is shipped with a node based command line tool. It might be installed as a global tool on your computer to render a mustache template of some kind
+
+```bash
+$ npm install -g mustache
+
+$ mustache datdataView.json myTemplate.mustache > output.html
+```
+
+also supports stdin.
+
+```bash
+$ mustache dataView.json | mustache - myTemplate.mustache > output.html
+```
+
+or as a package.json `devDependency` in a build process maybe?
+
+```json
```

Comment:
Here it's missing the install instructions for using the cli on the `package.json`, look at the original README.